frontend/src/modules/dealerships/DealershipLocations.vue

```diff
@@ -0,0 +1,345 @@
+<template>
+  <div>
+    <CCard class="mt-2">
+      <CCardBody>
+        <h3 class="mb-3">Dealership Zones</h3>
+        <CRow>
+          <CCol v-if="addingNewZone">
+            <CForm @submit.prevent="submitNewZone" v-if="addingNewZone">
+              <CInput
+                label="Name"
+                placeholder="Unique name for the zone"
+                v-model="newZoneName"
+                required
+              />
+              <CTextarea
+                label="Description (optional)"
+                placeholder="Enter the zone's description"
+                rows="5"
+                v-model="newZoneDescription"
+              />
+              <CButton
+                color="secondary"
+                @click="
+                  () => {
+                    path = [];
```

Comment:
This should be newPath and not path, because when I cancelled creating a zone, the zone stays on the map

Reply:
Addressed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/87/commits/b3016ee9df8271d80dc6fc49195c95de810e1398)